src/App.tsx

```diff
@@ -76,17 +81,16 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && isLoaded && !hasError && posts.length === 0 && (
```

Comment:
Use logical operators 
```suggestion
                {author && isLoaded && !hasError && !posts.length && (
```